docker-compose.yml

```diff
@@ -14,6 +14,7 @@ services:
       #     https://www.elastic.co/guide/en/x-pack/current/installing-xpack.html#xpack-enabling
       xpack.security.enabled: "false"
       xpack.monitoring.enabled: "false"
+      xpack.ml.enabled: "false"
```

Comment:
Should also be disabled in the [`kibana.yml`](https://github.com/deviantony/docker-elk/blob/52f43a4a72dbfef0a0fe604bad384027be7c9faa/kibana/config/kibana.yml) file for completeness. See https://www.elastic.co/guide/en/x-pack/current/installing-xpack.html#xpack-enabling